jinahub/encoders/audio/VGGISHAudioEncoder/vggish_audio_encoder.py

```diff
@@ -70,14 +108,15 @@ def _get_input_data(self, docs: DocumentArray, parameters: dict):
         filtered_docs = DocumentArray([doc for doc in flat_docs if doc.blob is not None])
 
         return filtered_docs
+
     def _create_embeddings(self, filtered_docs: Iterable):
         """Update the documents with the embeddings generated by VGGISH"""
 
         for d in filtered_docs:
             # Vggish broadcasts across different length audios, not batches
             [embedding] = self.sess.run([self.embedding_tensor], feed_dict={self.feature_tensor: d.blob})
             result = self.post_processor.postprocess(embedding)
-            d.embedding = (np.float32(result) - 128.) / 128.
+            d.embedding = np.mean((np.float32(result) - 128.) / 128., axis=0)
```

Comment:
is it possible we need other like max pooling methods?

Reply:
Thanks for the suggestion, for eap-qupeiyin, we are using average pooling. To add other options such as max pooling, I think we can leave this for the next PR.